README.md

```diff
@@ -15,12 +15,17 @@ Things you may want to cover:
 ## Database creation
 
 ## Database initialization
+Once you have the proiject opened in VS Code. Install Postgres. Then Click the PostgresSQL Elephant icon, select the myapp_development dropdown menu, select the public drop down menu, right click users, hover over "Select..." and click "Run Select Top 1000". 
 
 ## How to run the test suite
 
+Feel free to add cucumber test cases to the cucumber test files and steps to the steps files. To run the current cucumber tests, enter the command "rails cucumber"
+
 ## Services (job queues, cache servers, search engines, etc.)
+The application utilizes Google Maps' search engine to return relevant venues near the user's zipcode. 
 
 ## Deployment instructions
+Deploy the rails app to heroku. 
```

Comment:
How? you cant just state deploy to heroku!

Refer to the wiki for instructions! you can probably just link it here!